api/src/main/java/org/apache/druid/indexer/TaskStatusPlus.java

```diff
@@ -44,13 +47,42 @@
   @Nullable
   private final String errorMsg;
 
-  @JsonCreator
   public TaskStatusPlus(
       @JsonProperty("id") String id,
       @JsonProperty("type") @Nullable String type, // nullable for backward compatibility
       @JsonProperty("createdTime") DateTime createdTime,
       @JsonProperty("queueInsertionTime") DateTime queueInsertionTime,
-      @JsonProperty("statusCode") @Nullable TaskState state,
+      @JsonProperty("statusCode") @Nullable TaskState statusCode,
```

Comment:
This constructor isn't `@JsonCreator` so let's get rid of the `@JsonProperty` annotations here

Reply:
right, will remove the annotations.